awx/main/models/credential.py

```diff
@@ -369,6 +374,31 @@ def display_inputs(self):
                 field_val[k] = '$encrypted$'
         return field_val
 
+    def unique_hash(self, display=False):
```

Comment:
Does anything call `unique_hash(display=True)`?  If not, do we need this extra logic?

Reply:
Yeah, I started doing that for the view (when returning error messages). I should document what the 2 look like, if we keep the dichotomy

display format: `vault (id=alan)`
other format: `3_alan` (assuming 3 is the pk of the vault type)

I'd vote that we should keep the display format if we're going to use long names like literally "Amazon Web Services".

Reply:
Okay, so integration tests have told me what the expectation is (`test_confirm_only_single_managed_by_tower_extra_credential_allowed`). They are expecting the error message:

```python
{'error': 'Cannot assign multiple VMware vCenter credentials.'}
```

This is pre-existing in 3.2, so easiest to just keep it the same.